packages/menu/demo/stories/MenuStory.tsx

```diff
@@ -67,6 +93,8 @@ const Component = ({
   return (
     <div className="relative">
       <button {...getTriggerProps()}>Produce</button>
```

Comment:
```suggestion
      <button className="px-2 py-1" type="button" {...getTriggerProps()}>Produce</button>
```

While we're here, it would be nice to get this up-to-par with `Modal` and other stories :wink: